package.json

```diff
@@ -13,7 +13,7 @@
     "test": "jest",
     "test:unit": "jest --testPathIgnorePatterns test/git.integration-test.js",
     "test:git-integration": "jest --testPathPattern test/git.integration-test.js",
-    "release": "bin/cli.js"
```

Comment:
I hadn't noticed this - but we don't actually use this script to release any more - it could be removed.

(commit-and-tag-version is actually released with release please - very convenient for github projects, if your use-case is covered and you don't run into the many bugs it has)

Reply:
Sure. No problem. I can remove it when I return to this PR (hopefully this weekend)

For convenience I actually use it to debug the changes